Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -103,7 +103,7 @@ import BraintreeCore
     /// - Warning: This initializer should be used for merchants using the PayPal App Switch flow. This feature is currently in beta and may change or be removed in future releases.
     /// - Note: The PayPal App Switch flow currently only supports the production environment.
     public convenience init(
-        userAuthenticationEmail: String,
+        userAuthenticationEmail: String? = nil,
```

Comment:
Do we want to consider removing the convenience inits?

Reply:
sure, can maybe address this in follow-up pr or ticket.